modules/ROOT/pages/message-flows.adoc

```diff
@@ -19,55 +19,71 @@ Send messages to trading partners. An outbound message flow receives application
 
 Create an inbound message flow to receive EDI transactions from your trading partner, transform, and then send the translated application message to your backend systems.
 
-The following diagram shows the inbound message configuration building blocks of an inbound message flow and how the Partner Manager’s message processing framework orchestrates the end-to-end routing and transformation of your inbound EDI transactions from your trading partners to your backend systems at runtime.
+The following diagram shows the inbound message configuration of an inbound message flow. The diagram also shows how Partner Manager’s message processing framework orchestrates the end-to-end routing and transformation of your inbound EDI transactions from your trading partners to your backend systems at runtime.
 
 image::inbound-message-flow.png[Inbound message flow configuration building blocks diagram]
 
-. Receiving endpoint +
-Endpoint where you (the host) receive transmissions from the trading partner. Supported protocols include:
-* AS2 
+|===
+|Setting |Description |Supported Formats/Protocols
+
+|Receiving endpoint | The endpoint where the host receives transmissions from the trading partner. a|
+* AS2
 * SFTP
-. Source message type +
-Message type to process for this message flow, using the EDI X12 formats and EDI identifiers.
-. Transform +
-The DataWeave map for EDI to transform into XML or JSON formats.
-. Target message type +
-Supported formats include JSON and XML. 
-This is also the format into which DataWeave map transforms received messages.
-. Target endpoint +
-The target endpoint is the final destination of a message flow and is defined by a host message type and endpoint, often a host API. Supported protocols include:
+
+|Source message type |The message type to process for the message flow. a|
+* EDI X12 formats
+* EDI identifiers
+
+|Transform |The DataWeave map used to transform EDI into XML or JSON formats. a|
+* JSON
+* XML
+
+|Target message type |The format into which the DataWeave map transforms received messages. a|
+* JSON
+* XML
+
+|Target endpoint |The final destination of a message flow. The target endpoint is defined by a host message type and endpoint, often a host API. a|
 * SFTP
 * HTTP
 * HTTPS
 
+|===
+
 [[outbound-message-flow]]
 == Outbound Message Flows
```

Comment:
ditto edits to Inbound Message Flows

Reply:
GLOBALLY search for "back-end" and change it to "backend," per Term Presentation List.

Reply:
Done